nodejs/builder/Dockerfile.tmpl

```diff
@@ -0,0 +1,57 @@
+FROM unocha/nodejs-base:%%UPSTREAM%%
+
+# Parse arguments for the build command.
+ARG VERSION
+ARG VCS_URL
+ARG VCS_REF
+ARG BUILD_DATE
+
+# A little bit of metadata management.
+# See http://label-schema.org/
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vendor="UN-OCHA" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.vcs-url=$VCS_URL \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.name="alpine-nodejs-builder" \
+      org.label-schema.description="This service provides a builder container for node." \
+      org.label-schema.architecture="x86_64" \
+      org.label-schema.distribution="Alpine Linux" \
+      org.label-schema.distribution-version="3.6" \
+      info.humanitarianresponse.node="6.12.0" \
+      info.humanitarianresponse.npm="3.10.10" \
+      info.humanitarianresponse.ruby="2.4.2-r0"
+
+ENV DST_DIR=/dst
+
+ENTRYPOINT []
+
+RUN apk update && \
+    apk upgrade && \
+    apk add \
+        rsync \
+        ruby-bundler \
+        build-base \
+        ruby-dev \
+        libffi-dev && \
+    gem install --no-document \
+        bootstrap-sass \
+        font-awesome-sass \
+        sass-globbing \
+        compass && \
+    mkdir -p /root/.npm && \
```

Comment:
This was a hack to avoid an npm problem, but since we're using yarn now to install, there's no need to create this dir (and remove it on line 52).

Reply:
true. will remove it.